src/DynamoCore/Migration/Migration.cs

```diff
@@ -1360,6 +1354,7 @@ public class NodeMigrationAttribute : Attribute
         /// <summary>
         /// Version this migrates to.
         /// </summary>
+        [Obsolete("All migrations are now processed, beginning with the first migration whose version is >= the workspace version. The To property will be ignored.", false)]
```

Comment:
hmm, if this is targeted towards Dynamo 2.0 release should we just get rid of this? Or you prefer the tag for documentation purposes?